src/devices/Arduino/FirmataCommandSequence.cs

```diff
@@ -11,26 +11,71 @@ namespace Iot.Device.Arduino
     /// A firmata command sequence
     /// Intended to be changed to public visibility later
     /// </summary>
-    internal class FirmataCommandSequence
+    public class FirmataCommandSequence
     {
+        /// <summary>
+        /// Start of sysex command byte. Used as start byte for almost all extended commands.
+        /// </summary>
+        public const byte StartSysex = (byte)FirmataCommand.START_SYSEX;
```

Comment:
I'm missing something here, why is this public, why not just use  `FirmataCommand.START_SYSEX` wherever it's needed?

Reply:
Because FirmataCommand is not public (all the other members of that enum are not used by external components)